### PR TITLE
show custom data on status page

### PR DESCRIPTION
**Pull request by kelevro**
Show custom data properties on status page
screen shot 2018-06-04 at 6 38 12 pm

### CODE REVIEW

**Comment by kenaniah** [comment]
Thanks for the PR! I'd be happy to merge this (and release a new version) once the tests pass.

**Comment by kelevro** [comment]
@kenaniah great. i'll fix tests in a while. i have a lot of work now

File: lib/sidekiq-status/web.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ module Web
 
     DEFAULT_PER_PAGE_OPTS = [25, 50, 100].freeze
     DEFAULT_PER_PAGE = 25
+    COMMON_STATUS_HASH_KEYS = %w(update_time jid status worker args label pct_complete)
```
Comment (axos88) [inline lib/sidekiq-status/web.rb]: total, at, and message is missing

**Comment by rmm5t** [comment]
I came here looking for something like this. I was surprised that custom data wasn't already in the web view. 

@kelevro I know it's been a little while now since you worked on this, but were you planning to finish this PR? Do you want help on it? 

@axos88 If I helped get this feature green with a stacked PR on top of this one, and also addressed your comments, are you willing and available to merge and release a new version relatively soon afterwards?

**Comment by kenaniah** [comment]
@rmm5t yes and thanks!

**Comment by rmm5t** [comment]
@kenaniah Please see #146. It took me longer than I would like to admit to figure out what was causing the intermittent errors.

Thank you! 

I'm very much looking forward to a release with this feature. I have a client that could really benefit from it.

🍻

**Pull request closed without merging**